### DISCUSSION TITLE
UI Design for Timeloom

### DISCUSSION THREAD

**Discussion by inclinedadarsh** [inclinedadarsh/timeloom-old#1] [General]
Heyy everyone!
You can give feedback and suggestions for the UI Design of Timeloom. 

Things you can talk about:
- User Experience
- UI Design
- Consistency
- Better design components
- Idk what, anything you can relate with design

You can head to [UI Design Wiki](https://github.com/inclinedadarsh/timeloom/wiki/UI-Design) for Design updates.

**Comment by tiwariParth** [comment]
Hey team timeloom! You can take this as a suggestion if this is on purpose 
![image](https://github.com/inclinedadarsh/timeloom/assets/99398053/c2eafa34-ed07-480a-a467-fcb20c27b8a3)
The text on the left side is not aligned with the button below, if this on purpose centring it would make it look more cool i think. Would love to know what you guys think on it.
